siad/daemon.go

```diff
@@ -57,6 +57,10 @@ func processModules(modules string) (string, error) {
 func processConfig(config Config) (Config, error) {
 	var err error
 	config.Siad.APIaddr = processNetAddr(config.Siad.APIaddr)
+	apiaddr := modules.NetAddress(config.Siad.APIaddr)
+	if apiaddr.Host() != "" && !apiaddr.IsLoopback() && !config.Siad.AllowAPIBind {
+		return Config{}, errors.New("ERROR: You must pass --disable-security to bind Siad to a non-localhost address.")
```

Comment:
errors should not start with a capital or end in punctuation. The `ERROR:` is also redundant.